samples/wifi/throughput/README.rst

```diff
@@ -0,0 +1,308 @@
+.. _wifi_throughput_sample:
+
+Wi-Fi: Throughput
+#################
+
+.. contents::
+   :local:
+   :depth: 2
+
+The Throughput sample demonstrates how to measure the throughputs for Nordic Semiconductor's Wi-Fi® chipsets.
+
+Requirements
+************
+
+The sample supports the following development kit:
+
+.. table-from-sample-yaml::
+
+Overview
+********
+
+The sample demonstrates how to measure network performance using **zperf** tool in different profiles.
+Using this sample, the development kit can associate with an access point.
+
+Building and running
+********************
+
+.. |sample path| replace:: :file:`samples/wifi/throughput`
+
+.. include:: /includes/build_and_run_ns.txt
+
+Currently, the nRF7002 DK + QSPI configuration is supported.
```

Comment:
This could be made as a note. Also, make it more descriptive, e.g.

"The sample is supported on the nRF7002 DK with QSPI as the interface between the nRF5340 host and the nRF7002 device."